packages/block-library/src/categories/index.php

```diff
@@ -94,6 +94,24 @@ function register_block_core_categories() {
 	register_block_type(
 		'core/categories',
 		array(
+			'attributes'      => array(
+				'align'             => array(
+					'type' => 'string',
+					'enum' => array( 'left', 'center', 'right', 'wide', 'full' ),
```

Comment:
The JS and the PHP don't communicate about this right now, but one thing to note is that by specifying the `enum` here, but not explicitly listing out the alignments in the JS `supports.align`, it could be possible that new alignment options are added to the editor but aren't supported here. I think that's not a huge deal, but worth noting.